TACC/Core-Styles
# fix: tup-584 simpler tag styles

## Overview

Simpler `c-tag` styles. Let client handle details.

## Related

- [TUP-584](https://tacc-main.atlassian.net/browse/TUP-584)

## Changes

- **removed** `c-tag` styles
- **changed** `c-tag` styles

## Testing

1. Open http://localhost:3000/components/detail/c-tag--types.
2. Verify tag is less bold and padding is different.

## UI

| state | screenshot |
| - | - |
| before |  ![before](https://github.com/TACC/Core-Styles/assets/62723358/e5575351-ca9a-42c1-8144-80d216c6175f) |
| after | ![after](https://github.com/TACC/Core-Styles/assets/62723358/33ed6c93-59a7-4ccf-8da3-e0a14281f285) |
